app/controllers/thredded/private_topics_controller.rb

```diff
@@ -56,10 +57,13 @@ def private_topic
     def new_private_topic_params
       params
         .require(:private_topic)
-        .permit(:title, :locked, :sticky, :content, user_ids: [], category_ids: [])
+        .permit(:title, :locked, :sticky, :content, :user_ids, user_ids: [], category_ids: [])
         .merge(
           user: thredded_current_user,
-          ip: request.remote_ip)
+          ip: request.remote_ip).tap do |p|
```

Comment:
Could we move the `)` and everything after on that line down to the next line? There's a bit of visual/mental parsing needed here to catch that the merge ends and a tap begins